lib/ui/screens/home_screen/home_screen.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:flutter/material.dart';
+
+class HomeScreen extends StatelessWidget {
+  const HomeScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const Scaffold();

```

Comment:
何の画面かはわかるようにすると良さそう！

```suggestion
    return const Scaffold(
      body: Center(
        child: Text("Home"),
      ),
    );
```